use rattler

danielnachun opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

danielnachun committed:
use rattler

danielnachun committed:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.12.06.14.27.19

Other tools:
- conda-build 25.11.0
- rattler-build 0.51.0
- rattler-build-conda-compat 1.4.10

danielnachun committed:
use rattler

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

danielnachun committed:
don't cross-compile

danielnachun committed:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.12.08.07.28.33

Other tools:
- conda-build 25.11.0
- rattler-build 0.51.0
- rattler-build-conda-compat 1.4.10

conda-forge-admin commented:
Hi! This is the friendly conda-forge automerge bot!

I considered the following status checks when analyzing this PR:
 - **linter**: passed
 - **azure**: passed


Thus the PR was passing and merged! Have a great day!
